File: 0501_0600/0528/0528.py
with open("input.txt", "r") as f, open("output.txt", "w") as q:
    n, k = (int(x) for x in f.read().split())
    q.write(str(n + (2 + (2 + k) * (n - 2)) * (k - 1) // 2))

File: 0501_0600/0528/0528.go
package main

import (
	"fmt"
	"os"
)

func main() {
	f, _ := os.Open("input.txt")
	q, _ := os.Create("output.txt")
	var n, k int64
	fmt.Fscan(f, &n, &k)
	fmt.Fprint(q, n+(2+(2+k)*(n-2))*(k-1)/2)
}
